x-pack/plugins/cases/public/common/mock/test_providers.tsx

```diff
@@ -43,11 +44,21 @@ const TestProvidersComponent: React.FC<TestProviderProps> = ({
   userCanCrud = true,
   releasePhase = 'ga',
 }) => {
+  const queryClient = new QueryClient({
```

Comment:
making a test queryClient as part of the test providers to test components that use react-query.